tools-local/prebuilt-baseline-offline.xml

```diff
@@ -78,90 +84,39 @@
     <PackageIdentity Id="runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl" Version="4.3.2" />
     <PackageIdentity Id="runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl" Version="4.3.0" />
     <PackageIdentity Id="runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl" Version="4.3.2" />
-    <PackageIdentity Id="System.Collections.Immutable" Version="5.0.0-preview.8.20371.14" />
     <PackageIdentity Id="System.Composition" Version="1.0.31" />
-    <PackageIdentity Id="System.Reflection.Metadata" Version="5.0.0-preview.8.20371.14" />
-    <PackageIdentity Id="System.Runtime.CompilerServices.Unsafe" Version="5.0.0-preview.4.20202.18" />
-    <PackageIdentity Id="System.Text.Encodings.Web" Version="5.0.0-preview.4.20202.18" />
-    <PackageIdentity Id="System.Text.Json" Version="5.0.0-preview.4.20202.18" />
+    <PackageIdentity Id="System.Text.Encodings.Web" Version="4.7.1" />
     <PackageIdentity Id="System.Xml.XPath.XmlDocument" Version="4.0.1" />
     <PackageIdentity Id="XliffTasks" Version="1.0.0-beta.20420.1" />
   </NeverRestoredTarballPrebuilts>
   <Usages>
-  <!-- portable-build only prebuilts -->
-    <Usage Id="runtime.any.System.Collections" Version="4.3.0" Rid="any" />
-    <Usage Id="runtime.any.System.Diagnostics.Tools" Version="4.3.0" Rid="any" />
-    <Usage Id="runtime.any.System.Diagnostics.Tracing" Version="4.3.0" Rid="any" />
-    <Usage Id="runtime.any.System.Globalization" Version="4.3.0" Rid="any" />
```

Comment:
Not sure if it'll cause a problem to remove this manually maintained section, but pointing it out.

Reply:
Augh.  Missed that one.  Since the portable build is almost done, I'll wait and see if any of the portable prebuilts change and update it if so.